sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -683,6 +699,26 @@ describe('<item-list>', () => {
         assert(columns[0].innerText === testFixture.rows[sortedOrder[i]].columns[0].toString());
       }
     });
+
+    it('returns the correct selectedIndices result matching clicked items', () => {
```

Comment:
Should there be a test that includes sorting?

Reply:
This one does. It sorts the list first, reversing the order of the items 3 -> 2 -> 1, then selects the first one and makes sure its selected index is 2 not 0.

Reply:
I can add one more for filtering.